lib/shoryuken/manager.rb

```diff
@@ -3,7 +3,7 @@ class Manager
     include Util
 
     BATCH_LIMIT        = 10
-    HEARTBEAT_INTERVAL = 0.1
+    HEARTBEAT_INTERVAL = 1
```

Comment:
That's probably going to cause issues with empty queues as it will take a least `HEARTBEAT_INTERVAL` before it will request new messages.

Instead of this you could just have `dispatch` run in a loop and rely on the `TimerTask` to restart it if it ever dies.